tests/test_conan_version_bump.py

```diff
@@ -0,0 +1,18 @@
+import pytest
```

Comment:
Please add our header to our own code like in  https://github.com/nexB/univers/blob/1e56074798ca34f41d5d3442452c0b5029b4cd26/tests/test_version_range.py#L1

Reply:
It's not our own code, this is also coming from conan.